src/biokbase/narrative/jobs/job.py

```diff
@@ -79,8 +82,8 @@
 STATE_ATTRS = list(set(JOB_ATTRS) - set(JOB_INPUT_ATTRS) - set(NARR_CELL_INFO_ATTRS))
 
 
-class Job(object):
-    _job_logs = list()
+class Job:
+    _job_logs = []
```

Comment:
from the [flake8-comprehensions plugin](https://github.com/adamchainz/flake8-comprehensions) (installed locally)